pkgs/by-name/ai/airdrop-cli/package.nix

```
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  swift,
+  swiftpm,
+}:
+
+stdenv.mkDerivation {
+  pname = "airdrop-cli";
+  version = "unstable-2024-04-13";
```

```suggestion
  version = "0-unstable-2024-04-13";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning
Also, change the pull request title and commit message to this version

---

Fixed 👍